lib/kino_flame/runner_cell.ex

```diff
@@ -77,16 +153,36 @@ defmodule KinoFLAME.RunnerCell do
 
   defp to_updates(field, value), do: %{field => value}
 
+  @default_keys ["backend", "name", "min", "max", "max_concurrency"]
+
   @impl true
-  def to_attrs(ctx) do
+  def to_attrs(%{assigns: %{fields: fields, runner_pod_template: runner_pod_template}} = ctx) do
+    backend_keys =
+      case fields["backend"] do
+        "k8s" ->
+          []
+
+        "fly" ->
+          ~w|fly_cpus fly_cpu_kind fly_memory_gb fly_gpu_kind fly_gpus fly_envs|
+      end
+
     ctx.assigns.fields
+    |> Map.take(backend_keys ++ @default_keys)
+    |> Map.put("runner_pod_template", runner_pod_template)
```

Comment:
We want to only include the template if k8s:

```suggestion
    fields = Map.put(fields, "runner_pod_template", runner_pod_template)

    shared_keys = ["backend", "name", "min", "max", "max_concurrency"]

    backend_keys =
      case fields["backend"] do
        "k8s" ->
          ~w|runner_pod_template|

        "fly" ->
          ~w|fly_cpus fly_cpu_kind fly_memory_gb fly_gpu_kind fly_gpus fly_envs|
      end

    Map.take(fields, shared_keys ++ backend_keys)
```

Reply:
Also, let's rename `runner_pod_template` to `k8s_pod_template`, so it's clear this attribute is backend specific :D